docs/topics/native/native-ios-integration.md

```diff
@@ -244,6 +245,26 @@ It requires two GC cycles to collect these four objects because deinitialization
 after the GC cycle. The limitation stems from `deinit`, which can call arbitrary code, including the Kotlin code that
 cannot be run during the GC pause.
 
+### Retain cycles
+
+In a retain cycle, a number of objects refer each other using strong references cyclically:
+
+![Retain cycles](native-retain-cycle.png){height=200}
+
+Kotlin's GC and Objective-C's ARC handle retain cycles differently. When objects become unreachable, the GC can properly
+reclaim such cycles, while Objective-C's ARC can't. Therefore, retain cycles of Kotlin objects can be reclaimed,
+while retain cycles of Swift/Objective-C objects cannot.
```

Comment:
```suggestion
Kotlin's tracing GC and Objective-C's ARC handle retain cycles differently. When objects become unreachable, tracing GC can properly
reclaim such cycles, while ARC can't. Therefore, retain cycles of Kotlin objects can be reclaimed,
while retain cycles of Swift/Objective-C objects cannot.
```

How about this? Once again, spell out *tracing* GC.

Reply:
Maybe it also makes sense to link to https://docs.swift.org/swift-book/documentation/the-swift-programming-language/automaticreferencecounting/#Strong-Reference-Cycles-Between-Class-Instances ?